src/main/java/com/jhuguet/sb_taskv1/app/services/impl/CustomUserDetailsServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.jhuguet.sb_taskv1.app.services.impl;
+
+import com.jhuguet.sb_taskv1.app.models.User;
+import com.jhuguet.sb_taskv1.app.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+
+@Service
+public class CustomUserDetailsServiceImpl implements UserDetailsService {
```

Comment:
Why don't you encode user password anywhere?

Reply:
Ease of tests, future encoding will take place